src/app/modules/main/node_section/controller_interface.nim

```diff
@@ -0,0 +1,36 @@
+import json
+import status/types/[rpc_response]
+
+type 
+  AccessInterface* {.pure inheritable.} = ref object of RootObj
+  ## Abstract class for any input/interaction with this module.
+
+method delete*(self: AccessInterface) {.base.} =
+  raise newException(ValueError, "No implementation available")
+
+method init*(self: AccessInterface) {.base.} =
+  raise newException(ValueError, "No implementation available")
+
+method sendRPCMessageRaw*(self: AccessInterface, inputJSON: string): string {.base.} =
+  raise newException(ValueError, "No implementation available")
+
+method setBloomFilterMode*(self: AccessInterface, bloomFilterMode: bool): StatusGoError {.base.} =
```

Comment:
@borismelnik `RpcResponse` doesn't contain `StatusGoError` remove it should not be used in this context. State for all places.

Reply:
got it